packages/button/src/utils/getStyles.ts

```diff
@@ -191,7 +128,7 @@ export const getItemInteractiveStyles = ({
 }: IButtonProps): FlattenInterpolation<any> => {
   const standardFocus = css`
     &:focus {
-      box-shadow: 0 0 0 2px ${buttonTheme(theme, appearance, baseAppearance, 'boxShadowColor')};
+      box-shadow: 0 0 0 2px ${buttonAppearanceTheme(theme, appearance, baseAppearance, 'boxShadowColor')};
```

Comment:
I think box-shadow property must be themed, not just boxShadowColor.
This will be more flexible: `box-shadow: ${buttonAppearanceTheme(theme, appearance, baseAppearance, 'boxShadow')};`